x/blob/client/testutil/integration_test.go

```diff
@@ -123,12 +124,9 @@ func (s *IntegrationTestSuite) TestSubmitWirePayForBlob() {
 			s.Require().NoError(s.network.WaitForNextBlock())
 
 			// attempt to query for the malleated transaction using the original tx's hash
-			qTxCmd := authcmd.QueryTxCmd()
-			out, err = clitestutil.ExecTestCLICmd(clientCtx, qTxCmd, []string{txResp.TxHash, "--output=json"})
+			res, err := testfactory.QueryWithoutProof(clientCtx, txResp.TxHash)
 			require.NoError(err)
-
-			var result sdk.TxResponse
-			s.Require().NoError(val.ClientCtx.Codec.UnmarshalJSON(out.Bytes(), &result))
+			require.Equal(abci.CodeTypeOK, res.TxResult.Code)
```

Comment:
we have to use the normal query mechanism, and not the this one because the cosmos-sdk mocks the inputs and outputs, which doesn't know how to decode a blob transaction.